test/extended/testdata/build-pruning/errored-build-config.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  name: myphp
+  labels:
+    app: myphp
+  annotations:
+    openshift.io/generated-by: OpenShiftWebConsole
+spec:
+  failedBuildsHistoryLimit: 2
+  triggers: {}
+  runPolicy: Serial
+  source:
+    type: Git
+    git:
+      uri: 'https://github.com/openshift/cakephp-ex.git'
+      ref: master
+  strategy:
+    type: Source
+    sourceStrategy:
+      env:
+        - name: FIELDREF_ENV
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.nofield
+      from:
+        kind: ImageStreamTag
+        namespace: openshift
+        name: 'php:7.0'
+  output:
+    to:
+      kind: ImageStreamTag
+      name: 'myphp:latest'
```

Comment:
delete the output, it's not needed right?  (won't even try to push there since the build errors out)